integration-tests/contract-fetch-multiple-big-map-keys.spec.ts

```diff
@@ -32,6 +32,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
                 }
             });
             const contract = await op.contract();
+            console.log("contractAddress : "+ contract.address);
```

Comment:
I think you forgot to remove the console.log here

Reply:
fixed